### ISSUE TITLE
Fails with numeric keys

### ISSUE THREAD

**Issue by GeorgeA92** [Nykakin/chompjs#20]
`chompjs.parse_js_object()` fails with keys as integer.

Reproducible exaple (actual output - `JSONDecodeError`):
```python
>>> from chompjs import parse_js_object as pjs
>>> str_1 = "__INITIAL_STATE__={'categoryMap':{2:{'is_active':true}, 5:{'is_active':false}}}"
>>> pjs(str_1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\User\Anaconda2\envs\sh_edge_38_s241\lib\site-packages\chompjs\chompjs.py", line 37, in parse_js_object
    return json.loads(parsed_data, **json_params)
  File "C:\Users\User\Anaconda2\envs\sh_edge_38_s241\lib\json\__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "C:\Users\User\Anaconda2\envs\sh_edge_38_s241\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\User\Anaconda2\envs\sh_edge_38_s241\lib\json\decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 17 (char 16)
>>> try:
...     result = pjs(str_1)
... except Exception as e:
...    exc = e
>>> exc
JSONDecodeError('Expecting property name enclosed in double quotes: line 1 column 17 (char 16)')
>>> exc.doc
'{"categoryMap":{2:{"is_active":true},5:{"is_active":false}}}'
```
Expected output - js object loaded as dict without errors.
In Chrome->Dev tools -> Elements -> Javascript console.
`JSON.stringify` ([V8 source](https://github.com/v8/v8/blob/9.2.45/src/json/json-stringifier.cc)) returns valid JSON with integer keys converted into str.
![image](https://user-images.githubusercontent.com/15800727/114943595-e580e700-9e4e-11eb-8799-232482ecde0d.png)
```python
>>> from json import loads
>>> expected_output = loads('{"categoryMap":{"2":{"is_active":true},"5":{"is_active":false}}}')
>>> expected_output
{'categoryMap': {'2': {'is_active': True}, '5': {'is_active': False}}}
>>> type(expected_output)
<class 'dict'>
```

**Comment by noviluni** [comment]
I add here a simpler example causing the same issue to make the patch easier:

```python
>>> chompjs.parse_js_object('{123: "aaa"}')
```

**Comment by noviluni** [comment]
BTW, seems obvious but just in case, the result should be `{123: "aaa"}` (the dict key should be a number)

**Comment by Nykakin** [comment]
Partially resolved by #28, available in version 1.1.14
